book/src/merge-migration.md

```diff
@@ -43,11 +43,21 @@ present in post-merge blocks. Two new flags are used to configure this connectio
   `http://localhost:8551`.
 - `--execution-jwt <FILE>`: the path to the file containing the JWT secret shared by Lighthouse and the
   execution engine.
+- `--execution-jwt-secret-key <STRING>`: the hex-encoded JWT secret shared by Lighthouse and the
```

Comment:
The paragraph above this states "Two new flags..." although we now have three options.

Considering that JWT file has been widely adopted as "normal" across CLs and ELs, I'd be tempted to leave this section as-is and perhaps add a `>`-indented tip at the bottom of the "Connecting to an execution engine" section like this:

> Tip: the `--execution-jwt-secret-key <STRING>` flag can be used instead of `--execution-jwt <FILE>`. This is useful, for example, for users who wish to inject the value into a Docker container without needing to write to a file.

If you decide to go with this suggestion you may not need any of the other changes in this file, either.

Reply:
Thanks for your review! I agree to all of your recommendations and have updated the code.
I was unsure about the `.to_string()` on the error message itself but I kept it for consistency.